### PR TITLE
Sle 15 sp4

### PR DESCRIPTION
**Pull request by varkoly**
## Problem
 - Default for xs_routing_mode was changed in HANA-2 SP7 (bsc#1216539)
  Set 'ports' as default for xs_routing_mode. This setting is backwards compatible.
- Append '/' after directories in find to be able to handle
  links to directories
- SAP Business One password policies do not match with HANA password policies (bsc#1215107)
  Implementing administrator passwort check in a script.
  If sap_bone pattern is installed apply SAP Bone password policies
  for the setup HANA also because this password will be used for
  the SAP Bone installation.
- bone-installation-wizard causes problems in the module definitions
  (bsc#1214821)
  Remove bad Provides and Obsoletes
- Code clean up:
    Use the same parameter in all scripts.
    Remove trailing spaces.
    Use $() instead of backticks.
    Remove derprecated parameter -g
- Adapt pattern name (bsc#1214161)
  SAP installation wizard stops after HANA installation

### CODE REVIEW

File: src/lib/y2sap/auto.rb [hunk 1]
```diff
@@ -177,7 +177,7 @@ def prepare_hana(prod)
         return :abort
       end
       !prod.key?("sapMDC") || prod["sapMDC"] = "no"
-      File.write(@inst_dir + "/ay_q_masterPwd", prod["sapMasterPW"])
+      File.write(@inst_dir + "/ay_q_masterPwd",  prod["sapMasterPW"])
```
Comment (joseivanlopez) [inline src/lib/y2sap/auto.rb:180]: NP: double space before `prod[...`. Is that intentional?

File: package/sap-installation-wizard.spec [hunk 2]
```diff
@@ -19,7 +19,7 @@ Name:           sap-installation-wizard
 Summary:        Installation wizard for SAP applications
 License:        GPL-2.0+
 Group:          System/YaST
-Version:        4.4.9
+Version:        4.4.12
```
Comment (joseivanlopez) [inline package/sap-installation-wizard.spec:22]: Why jumping 3 versions?

Comment (varkoly) [reply package/sap-installation-wizard.spec:22]: We have developed SAPBone for SLES-SP4. This have be made in the "hot phase" very rapidly and I had to reackt very quickly. As result 4.4.11 is already released in SUSE:SLE-15-SP4:Update:Products:SAPBOne:Update/sap-installation-wizard.

File: src/lib/y2sap/auto.rb (hunk 1 above)
Comment (varkoly) [reply src/lib/y2sap/auto.rb]: Too much readability :-)

**Review by joseivanlopez — APPROVED** [review]

**Pull request merged**